Babylon (#80)
Joystream/helpdesk: roles/content-curators/README.md

"Speak softly, and carry a big stick"
- Theodore Roosevelt

## Curation

> Curation can mean either censoring, updating or altogether removing content.

Reviewer: ```suggestion
Curation can mean either censoring, updating or altogether removing content. It can also involve deciding which content should be "featured" on the platform.
```

A key part of understanding Curation is to understand the basics of the Content Directory using the [CLI](https://github.com/Joystream/joystream/tree/master/cli). Examples:

```
# Info about a specific class:
$ joystream-cli content-directory:class <CLASSNAME|CLASSID>

# Info about a specific entity:
$ joystream-cli content-directory:entity <ENTITYID>

# List all classes:
$ joystream-cli content-directory:classes

# List all entities in a class:
$ joystream-cli content-directory:entities <CLASSNAME|CLASSID>
```

### Censoring

> The first step to perform if a video or channel is not in line with the Terms is to censor it.

Reviewer: ```suggestion
The first step to perform if a video or channel is not in line with the testnet Terms of Service is to censor it.
```

> If the Curator has "Maintainer" permsission for a Channel or Video, this is done by the following [CLI](https://github.com/Joystream/joystream/tree/master/cli) command `joystream-cli media:curateContent`. Suppose you want to censor a Video with `entityId` 300:

Reviewer: ```suggestion
If the Curator has "Maintainer" permission for a Channel or Video, this is done by the following [CLI](https://github.com/Joystream/joystream/tree/master/cli) command `joystream-cli media:curateContent`. Suppose you want to censor a Video with `entityId` 300:
```

```
$ joystream-cli media:curateContent --className 10 --status Censored --id 300
```

> If it's a small infraction, you can try to reach out to the creator, and have them make the modification required. If the issue gets resolved, you can reverse the censoring by:

Reviewer: ```suggestion
If it's a minor issue, you can try to reach out to the creator, and have them make the modification required. If the issue gets resolved, you can reverse the censoring by:
```
